api/clients/disperser_client.go

```diff
@@ -273,18 +284,25 @@ func (c *disperserClient) GetBlobStatus(ctx context.Context, requestID []byte) (
 	return c.client.GetBlobStatus(ctxTimeout, request)
 }
 
-func (c *disperserClient) RetrieveBlob(ctx context.Context, batchHeaderHash []byte, blobIndex uint32) ([]byte, error) {
+func (c *DisperserClient) RetrieveBlob(ctx context.Context, batchHeaderHash []byte, blobIndex uint32) ([]byte, error) {
 	err := c.initOnceGrpcConnection()
 	if err != nil {
 		return nil, api.NewErrorInternal(err.Error())
 	}
 
 	ctxTimeout, cancel := context.WithTimeout(ctx, time.Second*60)
 	defer cancel()
-	reply, err := c.client.RetrieveBlob(ctxTimeout, &disperser_rpc.RetrieveBlobRequest{
-		BatchHeaderHash: batchHeaderHash,
-		BlobIndex:       blobIndex,
-	})
+	if c.config.MaxRetrieveBlobSizeBytes == 0 {
+		// max blob size on mainnet is currently 16MiB but we set this to 100MiB for backward compatibility
```

Comment:
@samlaf are you planning to update this comment every time the disperser's `MAX_BLOB_SIZE` config is changed? I would rephrase this to `set to 100MiB for forward compatibility. check official documentation for current max blob size on mainnet`

Reply:
Good call: [a07336c](https://github.com/Layr-Labs/eigenda/pull/849/commits/a07336cf2fce85b7d1cf886a3d07f3306ac459b8)